### PR TITLE
chore: bump sdk to v0.45.16-ics

### PR DESCRIPTION
**Pull request by smarshall-spitzbart**
## Description

Bumps sdk to https://github.com/cosmos/cosmos-sdk/releases/tag/v0.45.16-ics, which contains the fix from https://github.com/cosmos/cosmos-sdk/pull/16097

### Author Checklist

*All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.*

I have...

* [x] Included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
* [x] Added `!` to the type prefix if API or client breaking change
* [x] Targeted the correct branch (see [PR Targeting](https://github.com/cosmos/interchain-security/blob/main/CONTRIBUTING.md#pr-targeting))
* [x] Provided a link to the relevant issue or specification
* [x] Followed the guidelines for [building SDK modules](https://github.com/cosmos/cosmos-sdk/blob/main/docs/docs/building-modules)
* [ ] Included the necessary unit and integration [tests](https://github.com/cosmos/interchain-security/blob/main/CONTRIBUTING.md#testing)
* [ ] Added a changelog entry to `CHANGELOG.md`
* [ ] Included comments for [documenting Go code](https://blog.golang.org/godoc)
* [ ] Updated the relevant documentation or specification
* [x] Reviewed "Files changed" and left comments if necessary 
* [ ] Confirmed all CI checks have passed

### Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items.*

I have...

* [ ] confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
* [ ] confirmed `!` in the type prefix if API or client breaking change
* [ ] confirmed all author checklist items have been addressed 
* [ ] reviewed state machine logic
* [ ] reviewed API design and naming
* [ ] reviewed documentation is accurate
* [ ] reviewed tests and test coverage

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -151,7 +151,6 @@ require (
 )
 
 replace (
-	github.com/cosmos/cosmos-sdk => github.com/cosmos/cosmos-sdk v0.45.15-ics
```
Comment (smarshall-spitzbart) [inline go.mod:154]: This replace doesn't do anything now that we reference mainline sdk and not a special informal fork

Comment (faddat) [reply go.mod:154]: cool

Comment (faddat) [reply go.mod:154]: wait Khanh and I actually do know how to use Git?

:/

**Comment by smarshall-spitzbart** [comment]
Doesn't affect ICS

**Pull request closed without merging**